ds-dl-intro/who_with_quizlink.md

```diff
@@ -1,14 +1,33 @@
-Learners are expected to have the following knowledge:
+This course is for you if:
+
+You have been using machine learning but are starting to dive in to the field of deep learning and you want to learn:
+- What is deep learning?
+- How to structure a deep learning project?
+- What are the different techniques used in deep learning?
+- How to address common pitfalls in a deep learning workflow?
```

Comment:
- Prepare input data for use for deep learning
> - Design and train a Deep Neural Network
> - Troubleshoot the learning process
> - Measure the performance of the network
> - Visualizing data and results
> - Re-use existing network architectures with and without pre-trained weights

```suggestion
- What is deep learning?
- How to structure a deep learning project?
- How to prepare input data for use for deep learning?
- How to design and train a basic neural network using Keras?
- How to evaluate a neural network?
- How to address common pitfalls in a deep learning workflow?
```
I think this makes it a bit more specific